RecoLocalTracker/SiStripClusterizer/plugins/SiStripClusters2ApproxClusters.cc

```diff
@@ -27,25 +44,109 @@ class SiStripClusters2ApproxClusters : public edm::stream::EDProducer<> {
   edm::EDGetTokenT<edmNew::DetSetVector<SiStripCluster> > clusterToken;
 
   unsigned int maxNSat;
+  static constexpr double subclusterWindow_ = .7;
+  static constexpr double seedCutMIPs_ = .35;
+  static constexpr double seedCutSN_ = 7.;
+  static constexpr double subclusterCutMIPs_ = .45;
+  static constexpr double subclusterCutSN_ = 12.;
+
+  edm::InputTag beamSpot;
```

Comment:
actually the recommendation for data members is to use the trailing underscore: see rule 2.9 at [*].
Please use the trailing underscore for every data-member.

[*] 
https://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1